src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -13467,10 +13476,10 @@ void Player_Action_8084F390(Player* this, PlayState* play) {
 }
 
 void Player_Action_8084F608(Player* this, PlayState* play) {
-    if ((DECR(this->av2.actionVar2) == 0) && func_8083ADD4(play, this)) {
+    if ((DECR(this->av2.actionVar2) == 0) && Player_StartCsAction(play, this)) {
         func_80852280(play, this, NULL);
-        Player_SetupAction(play, this, Player_Action_80852E14, 0);
-        Player_Action_80852E14(this, play);
+        Player_SetupAction(play, this, Player_Action_CsAction, 0);
+        Player_Action_CsAction(this, play);
     }
 }
```

Comment:
Any particular thoughts on this Action or a possible name? It's also setting up a cutscene but can add a delay with `DECR(this->av2.actionVar2)`? And once that delay is done, then it calls `Player_StartCsAction`. And once `Player_StartCsAction` is successful, then the action is setup to be `Player_Action_CsAction`

Reply:
I havent put any thought into this no. Its weird that is does `Player_SetupAction` because its entirely redundant. `Player_StartCsAction` would have already done it